docs/multi-homing.md

```diff
@@ -234,6 +234,67 @@ spec:
 - specifying a static IP address for the pod is only possible when the
   attachment configuration does **not** feature subnets.
 
+## Multi-network Policies
+OVN-Kubernetes implements native support for
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
+an API providing
+[network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+features for secondary networks.
+
+To configure pod isolation, the user must:
+- provision a `network-attachment-definition`.
+- provision a `MultiNetworkPolicy` indicating to which secondary networks it
+  applies via the
+  [policy-for](https://github.com/k8snetworkplumbingwg/multi-networkpolicy#policy-for-annotation)
+  annotation.
+
+Please refer to the following example:
+```yaml
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: tenant-blue
+spec:
+    config: '{
+        "cniVersion": "0.4.0",
+        "name": "tenant-blue",
+        "netAttachDefName": "default/tenant-blue",
+        "topology": "layer2",
+        "type": "ovn-k8s-cni-overlay",
+        "subnets": "192.168.100.0/24"
+    }'
+---
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: tenant-blue # indicates the net-attach-defs this policy applies to
```

Comment:
should be default/tenant-blue in your case?

Reply:
Good point; done.